### DISCUSSION TITLE
What about SQL-like databases?

### DISCUSSION THREAD

**Discussion by iamgiolaga** [deta/cloud-docs#589] [Ideas]
Hi! 

First of all, congrats for the awesome work! Are you planning in the future to support relational DBs like PostgreSQL or MySQL, as well?

Thank you

**Comment by etrotta** [comment]
You can include a (read-only) SQLite database amongst your project's source files or use an external database made to work with serverless applications though HTTP requests, but there are no concrete plans to support SQL yet as far as I know. It is a frequently requested feature though.
